futures-util/Cargo.toml

```diff
@@ -21,15 +21,19 @@ alloc = ["futures-core-preview/alloc"]
 async-await = []
 compat = ["std", "futures_01"]
 io-compat = ["io", "compat", "tokio-io"]
-bench = []
-nightly = ["futures-core-preview/nightly"]
-cfg-target-has-atomic = ["futures-core-preview/cfg-target-has-atomic"]
 sink = ["futures-sink-preview"]
 io = ["std", "futures-io-preview", "memchr"]
 channel = ["std", "futures-channel-preview"]
 join-macro = ["async-await", "futures-join-macro-preview", "proc-macro-hack", "proc-macro-nested"]
 select-macro = ["async-await", "futures-select-macro-preview", "proc-macro-hack", "proc-macro-nested", "rand"]
 
+# Unstable features
+# These features are outside of the normal semver guarantees and require the
+# `unstable` feature as an explicit opt-in to unstable API.
+unstable = ["futures-core-preview/unstable"]
+cfg-target-has-atomic = ["futures-core-preview/cfg-target-has-atomic"]
+bench = []
```

Comment:
`"bench"` feature is a feature for exposing `BiLock`, but it is not a stable public API and should be included here.

Related: #1679

Reply:
(Maybe this should be renamed to `bilock`.)

Reply:
Filed #1827